azure-pipelines.yml

```diff
@@ -63,11 +63,7 @@ jobs:
         if ($branchName -eq "refs/heads/main") {
             Write-Host "##vso[task.setvariable variable=SignTypeToUse]real"
             Write-Host "##vso[task.setvariable variable=TelemetryType]TELEMETRY_PRODUCTION"
-            Write-Host "Done setting the sign type and telemetry key instance for staging branch"
```

Comment:
💡☕️ For Archival reasons, I believe this is set because we no longer want to keep the staging et. al. anymore?

Reply:
Previous team who owned B2K had a different CI/CD process, they would move code from main branch to staging branch and then to production branch. sometimes code to production was cherry picked, but we want cleaner approach. so whatever is in main branch will be released to dev --> staging --> production. Hence checking for staging/production branch checks are removed. I hope this make sense. Thank you.